index.d.ts

```diff
@@ -0,0 +1,50 @@
+export as namespace EventEmitter;
```

Comment:
Current definition of exports might work well with ES6 syntax but not so well with standard TypeScript import syntax: 

```javascript
import EventEmitter = require('eventemitter3');
new EventEmitter();
```
This code will not compile with current export definition and this syntax could be used by many users of this library that use it with TypeScript (especially in Node environment)